2200018/3456.py

```diff
@@ -0,0 +1,20 @@
+# 3456 직사각형 길이 찾기
+
+import sys
+from math import sqrt
+
+sys.stdin = open("input_3456.txt")
+
+T = int(input())
+
+for i in range(T):
+    ls = list(map(int, input().split()))
+    
+    a, b = max(ls), min(ls)
+    
+    ls.remove(a)
+    ls.remove(b)
+
+    c = ls[0]
+
+    print(f"#{i + 1} {int(sqrt(a**2 + b**2 - c**2))}")
```

Comment:
깔끔하네요! 👍 
```suggestion
    print(f"#{i + 1} {a+b-c}")
```
제곱을 사용하지 않고 출력하는 방법도 공유 드릴게요!